### PR TITLE
Incrementing dest

### PR DESCRIPTION
**Pull request by pyrat**
This is allowing messages to be sent with incrementing destination addresses for HLR testing.

### CODE REVIEW

File: cmd/smpperf.go [hunk 1]
```diff
@@ -29,6 +26,11 @@ var password = flag.String("p", "", "password of SMPP server")
 var host = flag.String("h", "127.0.0.1:2775", "host of SMPP server")
 var purge = flag.Bool("purge", false, "waits to receive any pending receipts")
 
+var mode = flag.String("mode", "static", "Mode of destination address (static or dynamic)")
+var dst = flag.Int("dst", 447582668509, "Destination address")
+var src = flag.String("src", "447582668506", "Source address")
+var verbose = flag.String("verbose", "false", "Be verbose")
```
Comment (makkalot) [inline cmd/smpperf.go]: you can use flag.Bool here

Comment (pyrat) [reply cmd/smpperf.go]: Ah nice

File: cmd/smpperf.go [hunk 2]
```diff
@@ -202,12 +217,19 @@ func sendMessages(numMessages int) {
 
 func main() {
 	flag.Parse()
+	messageText := strings.Join(flag.Args(), " ")
+	if len(messageText) > 0 {
+		// all good
```
Comment (makkalot) [inline cmd/smpperf.go:222]: empty if

Comment (pyrat) [reply cmd/smpperf.go:222]: i like empty ifs

Comment (makkalot) [reply cmd/smpperf.go:222]: don't you just want to check this : 

```
if len(messageText) <= 0 {
    messageText = "text"
}
```

**Comment by pyrat** [comment]
yep; that is nicer

**Pull request merged**